source/views/calendar/event-detail.js

```diff
@@ -34,6 +36,34 @@ function MaybeSection({header, content}: {header: string, content: string}) {
     : null
 }
 
+function Links({header, content}: {header: string, content: string}) {
+  const rawUrls = getUrls(content)
+  const links = Array.from(rawUrls)
+
+  return links.length
+    ? <Section header={header}>
+        {links.map(url =>
+          <Cell
+            key={url}
+            cellStyle="Title"
+            title={url}
+            accessory="DisclosureIndicator"
+            onPress={() => openUrl(url)}
+          />,
+        )}
+      </Section>
+    : null
+}
+
+function cleanDescription(desc: string) {
+  const description = fastGetTrimmedText(desc || '')
+  if (description == 'See more details') {
```

Comment:
If we're told to "see more details", we take the URL within the hyperlink and display it instead.